commands/src/api/camera.rs

```diff
@@ -0,0 +1,12 @@
+pub use lumeo_events::camera::Camera;
+use serde::{Deserialize, Serialize};
+
+#[derive(Serialize, Deserialize, Debug, Clone)]
+pub enum Request {
+    /// Discover cameras. Returned data can be incomplete if camera
+    /// requires authorization. For authorized access use `GetWithAuth`.
```

Comment:
Referring to API that's not yet been added. Best move this change to the next commit.

Reply:
Good catch! Can it be done using fixups?

Reply:
nope :(